evap/static/ts/src/sortable-form.ts

```diff
@@ -0,0 +1,105 @@
+declare const Sortable: typeof import("sortablejs");
+
+class FormsetOptions {
+    prefix: string = "form";
+    deleteCssClass: string = "delete-row";
+    deleteText: string = "remove";
+    addText: string = "add another";
+    added: null | ((arg: JQuery<HTMLTableRowElement>) => void) = null;
+    formTemplate: string | null = null;
+}
```

Comment:
How do you feel about the following?
```suggestion
interface FormsetOptions {
    prefix: string;
    deleteCssClass: string;
    deleteText: string;
    addText: string;
    added: (arg: JQuery<HTMLTableRowElement>) => void;
    formTemplate: string | null;
}
```
Using `interface`, we don't need to the default values; I also removed the `null |` for `added`, since we don't use it. What do you think?